### ISSUE TITLE
39.7 enable-dag-cache not working

### ISSUE THREAD

**Issue by alexvenom** [NebuTech/NBMiner#589]
Saw multiple issues closed showing this syntax, but for some reason HiveOS won't allow me to set it:
![Screen Shot 2021-11-03 at 14 45 24](https://user-images.githubusercontent.com/4620702/140164189-af22a319-1474-4fb5-8590-5df6f3623f9a.png)

I have tried `"enable-dag-cache": true` which worked, mining ETH all right, but then the 3060Ti started crashing every time it was supposed to switch to ZIL:
![Screen Shot 2021-11-03 at 13 59 23](https://user-images.githubusercontent.com/4620702/140165277-f814b585-f28f-4ed3-a6f5-1760dd58f94c.png)

Is my syntax wrong? How should it be?

**Comment by ravadac** [comment]
I notice your hashrate is dropping to 0.000 as well.  I'm experiencing the same thing.  My hang is always on GPU 0.  I also get a segfault in dmesg.   Can you verify you get that as well?

**Comment by Tomohamaru** [comment]
Yeah. I have the same problem as You. It took like 10 with 0mh in my case to restart the miner.

**Comment by twk4924123** [comment]
same problem 
now trying difference command , not sure will work or not

**Comment by twk4924123** [comment]
good news guys
"enable-dag-cache": 1
worked for me , no error switch to zil and switch back to eth 
success get share from zil
